docs/en/ingest-management/release-notes/release-notes-8.12.asciidoc

```diff
@@ -14,13 +14,137 @@
 
 This section summarizes the changes in each release.
 
+* <<release-notes-8.12.1>>
 * <<release-notes-8.12.0>>
 
 Also see:
 
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.12.1 relnotes
+
+[[release-notes-8.12.1]]
+== {fleet} and {agent} 8.12.1
+
+Review important information about {fleet-server} and {agent} for the 8.12.1 release.
+
+[discrete]
+[[breaking-changes-8.12.1]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and
+performance. Before you upgrade, review the breaking changes, then mitigate the
+impact to your application.
+
+[discrete]
+[[breaking-170270-8.12.1]]
+.Naming collisions with {fleet} custom ingest pipelines
+[%collapsible]
+====
+*Summary* +
+If you were relying on an ingest pipeline of the form `${type}-${integration}@custom` introduced in version 8.12.0 (for example, `traces-apm@custom`, `logs-nginx@custom`, or `metrics-system@custom`) you need to update your pipeline's name to include an `.integration` suffix (for example, `logs-nginx.integration@custom`) to preserve your expected ingestion behavior.
+
+*Details* +
+In version 8.12.0, {fleet} added new custom ingest pipeline names for adding custom processing to integration data streams. These pipeline names used patterns as follows:
+
+* `global@custom`
+* `${type}@custom` (for example `traces@custom`)
+* `${type}-${integration}@custom` (for example `traces-apm@custom`)
+* `${type}-${integration}-${dataset}@custom` pre-existing (for example `traces-apm.rum@custom`)
+
+However, it was discovered in {kibana-issue}175254[#175254] that the `${type-integration}@custom` pattern can collide in cases where the `integration` name is _also_ a dataset name. The clearest case of these collisions was in the APM integration's data streams, for example:
+
+* `traces-apm`
+* `traces-apm.rum`
+* `traces-apm.sampled`
+
+Because `traces-apm`` is a legitimate data stream defined by the APM integration (see the relevant https://github.com/elastic/integrations/blob/main/packages/apm/data_stream/traces/manifest.yml[manifest.yml] file), it incurred a collision of these custom pipeline names on version 8.12.0. For example:
```

Comment:
Remove extra backtick

```suggestion
Because `traces-apm` is a legitimate data stream defined by the APM integration (see the relevant https://github.com/elastic/integrations/blob/main/packages/apm/data_stream/traces/manifest.yml[manifest.yml] file), it incurred a collision of these custom pipeline names on version 8.12.0. For example:
```

Reply:
Nice catch! Fixed in [4f67194](https://github.com/elastic/ingest-docs/pull/864/commits/4f671946f714f222d7f19782d4ce763e2fc3837f)